### PR TITLE
tony's commit

### PR DESCRIPTION
**Pull request by tonytanaka**
please review my code! Not sure about the last one, about e-mail validation

### CODE REVIEW

File: tonytanaka/index.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  } else
+    for (let i = 2; i < num; i++) {
+      if (num % i === 0) {
+        return false;
+      }
+      return true;
+    }
+}
+console.log(isPrime(7));
+//console.log(isPrime(9));
```
Comment (ciamac-da) [inline tonytanaka/index.js]: Task 12 ==> 9 is not a prime. A prime number is any positive integer greater than 1, which is only evenly divisible by two divisors: itself and 1. So 9 is  divisible by 1 , itself(9) and ALSO 3, therefore the result should be false but you get true

File: tonytanaka/index.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    email.includes("@.") ||
+    email.includes(".@")
+    //  * e.g. "@" must have at least one character in front of it.
+    // haven't check this one
+  ) {
+    return false;
+  }
+  return true;
+}
+console.log(validateEmail("john@email.com"));
```
Comment (ciamac-da) [inline tonytanaka/index.js:188]: Task 13 ==> 
 "j@example.com" should be valid(true) but you get false!
"john.smith@email.com" should be valid(true) but you face false!

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: tonytanaka/index.js (hunk 2 above)
Comment (tonytanaka) [reply tonytanaka/index.js:188]: can you give me a hint to solve this? I don't want to loose more 2h on it...

File: tonytanaka/index.js (hunk 1 above)
Comment (tonytanaka) [reply tonytanaka/index.js]: for this one, I decided to put the Math.sqrt as a maximum value for the loop, starting with 2

File: tonytanaka/index.js (hunk 2 above)
Comment (ciamac-da) [reply tonytanaka/index.js:188]: Take a look at slack:)

Comment (tonytanaka) [reply tonytanaka/index.js:188]: thanks!!

**Review by ciamac-da — APPROVED** [review]

**Pull request merged**